packages/rum/src/domain/getSessionReplayLink.ts

```diff
@@ -0,0 +1,53 @@
+import type {
+  RumConfiguration,
+  RumSessionManager,
+  ViewContexts,
+  RumSession,
+  SessionReplayUrlQueryParams,
+} from '@datadog/browser-rum-core'
+import { getSessionReplayUrl } from '@datadog/browser-rum-core'
+import { isBrowserSupported } from '../boot/isBrowserSupported'
+
+export function getSessionReplayLink(
+  configuration: RumConfiguration,
+  sessionManager: RumSessionManager,
+  viewContexts: ViewContexts,
+  isRecordingStarted: boolean
+): string | undefined {
+  const session = sessionManager.findTrackedSession()
+  const sessionId = session ? session.id : 'no-session-id'
+  const queryParams: SessionReplayUrlQueryParams = {}
+
+  const errorType = getErrorType(session, isRecordingStarted)
+  if (errorType) {
+    queryParams.errorType = errorType
+  }
+
+  const view = viewContexts.findView()
+  if (view) {
+    queryParams.seed = view.id
+    queryParams.from = view.startClocks.timeStamp
+  }
+
+  return getSessionReplayUrl(configuration, sessionId, queryParams)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about passing directly `session`, `view` and `errorType` to `getSessionReplayUrl`?
it would avoid to declare query params outside of the link computation and would avoid to have `no-session-id` declared at several spots.